l10n_ch_base_bank/tests/test_bank.py

```diff
@@ -61,14 +80,18 @@ def test_bank_iban_foreign(self):
         self.assertEqual(bank_acc.acc_type, 'iban')
 
     def test_bank_ccp(self):
-        bank_acc = self.env['res.partner.bank'].create({
+        # test UI onchange methods
+        bank_acc = self.env['res.partner.bank'].new({
             'partner_id': self.partner.id,
             'acc_number': '46-110-7',
         })
+
         bank_acc.onchange_acc_number_set_swiss_bank()
+        # ccp also changed
+        bank_acc.onchange_ccp_set_acc_number()
 
-        self.assertEqual(bank_acc.bank_id, self.bank)
-        self.assertEqual(bank_acc.acc_number, 'Bank/CCP Camptocamp')
+        self.assertFalse(bank_acc.bank_id.id)
```

Comment:
here shouldn't we find still the bank with it's ccp number?

Onchange ccp should have set the field `bank_id` with 
``` python
self.env['res.bank'].search([('ccp', '=', ccp)], limit=1)
```